### ISSUE TITLE
doc: need to extend controllers with evented mixin

### ISSUE THREAD

**Issue by Leooo** [sir-dunxalot/ember-easy-form-extensions#19]
necessary for routing-events methods, otherwise there is an ember error: `can not create hook didTransition`

``` javascript
//initializers/add-evented.js
import Ember from 'ember';
export default {
  name: 'add-evented',
  initialize: function(container, app) {
    Ember.Controller.reopen(Ember.Evented, {
      //necessary for easy-form-extensions hook initializers/routing-events
    });
  }
};
```

**Comment by sir-dunxalot** [comment]
Thanks for the issue. This is with Ember `v1.11`?

**Comment by Leooo** [comment]
yes Ember 1.11

**Comment by sir-dunxalot** [comment]
This is fixed with the major upgrade for Ember 1.11+. [You can test the prerelease here](https://github.com/sir-dunxalot/ember-easy-form-extensions/releases/tag/1.0.0.beta). Please note the API for inputs has changed slightly from the original Easy Form syntax.

**Comment by sir-dunxalot** [comment]
This will be fixed by #37
